README.rst

```diff
@@ -16,7 +16,7 @@ Visit `Python Application Performance Monitoring with New Relic`_ to learn more.
 .. _New Relic APM: http://newrelic.com/application-monitoring
 .. _Python Application Performance Monitoring with New Relic: http://newrelic.com/python
 
-Usage
+Installation
```

Comment:
```
FAILED
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 20: Warning: Title underline too short.
    Installation
    -----
```

Reply:
Hi @bradleycamacho - it looks like this header is still to short to pass `twine check` 😞